test/response.spec.js

```diff
@@ -15,6 +15,9 @@
     beforeEach(function() {
       service = new AWS.Service({
         apiConfig: new AWS.Model.Api({
+          metadata: {
+            signingName: 'ocean-wave'
+          },
```

Comment:
Why is this necessary?

Reply:
I mention that in the commit body:
This required an update to test data setup in response.spec.js (since the requests are now being signed, but the signing was failing because the service name for the signer was ending up as undefined)

Basically, the service that was being setup by new AWS.service has config.signatureVersion set to "V4", therefore, these requests are now signed. However, signing requires a signingName to be set. I chose 'ocean-wave' as a test service name.